Makefile

```diff
@@ -507,9 +507,11 @@ govet:
     ./test/k8sT/... \
     ./tools/...
 
-ineffassign:
-	@$(ECHO_CHECK) ineffassign
-	$(QUIET) ineffassign .
+lint:
+	#TODO remove once Vagrant version is updated
```

Comment:
@joestringer I have one related PR https://github.com/cilium/packer-ci-build/pull/211. However, as packer PR removes ineffassign, so to avoid any issue, would love to remote this TODO and bump vagrant version in another PR (once https://github.com/cilium/packer-ci-build/pull/211 is merged)